_activities/magnetism.md

```diff
@@ -1,15 +1,12 @@
 ---
 layout: activity
+lang: en
```

Comment:
remove language - these files actually should not have a specified language because they are duplicated in the process of building the site, creating one file for each language.